frontend/src/bundles/common/components/base-layout/components/suggestion-for-subscribing.tsx

```diff
@@ -0,0 +1,95 @@
+import { differenceInDays } from 'date-fns';
+
+import { ComponentSize } from '~/bundles/common/enums/component-size.enum.js';
+import { AppRoute } from '~/bundles/common/enums/enums.js';
```

Comment:
I think we can combine it like this: `import { AppRoute, ComponentSize } from '~/bundles/common/enums/enums.js';`

Reply:
I fixed it. Thank you)